exampleService/native/src/impl/ExampleService.cpp

```diff
@@ -42,54 +45,53 @@ namespace example {
       }
 
-      ::ndk::ScopedAStatus ExampleService::getGpuTemperature(float* _aidl_return) {
-         *_aidl_return = gpuValue();
+      ::ndk::ScopedAStatus ExampleService::GetGpuTemperature(float* _aidl_return) {
+         *_aidl_return = CpuValue();
          return ndk::ScopedAStatus::ok();
       }
 
-      ::ndk::ScopedAStatus ExampleService::getAmbientTemperature(float* _aidl_return) {
-         *_aidl_return = ambientValue();
+      ::ndk::ScopedAStatus ExampleService::GetAmbientTemperature(float* _aidl_return) {
+         *_aidl_return = AmbientValue();
          return ndk::ScopedAStatus::ok();
       }
 
-      ::ndk::ScopedAStatus ExampleService::getAverageCpuTemperature(float* _aidl_return) {
-         *_aidl_return = averageCpuValue();
+      ::ndk::ScopedAStatus ExampleService::GetAverageCpuTemperature(float* _aidl_return) {
+         *_aidl_return = AverageCpuValue();
          return ndk::ScopedAStatus::ok();
       }
 
-      ::ndk::ScopedAStatus ExampleService::getAverageGpuTemperature(float* _aidl_return) {
-         *_aidl_return = averageGpuValue();
+      ::ndk::ScopedAStatus ExampleService::GetAverageGpuTemperature(float* _aidl_return) {
+         *_aidl_return = AverageGpuValue();
          return ndk::ScopedAStatus::ok();
       }
 
-      ::ndk::ScopedAStatus ExampleService::getAverageAmbientTemperature(float* _aidl_return) {
-         *_aidl_return = averageAmbientValue();
+      ::ndk::ScopedAStatus ExampleService::GetAverageAmbientTemperature(float* _aidl_return) {
+         *_aidl_return = AverageAmbientValue();
          return ndk::ScopedAStatus::ok();
       }
 
-      ::ndk::ScopedAStatus ExampleService::getMaxCpuTemperature(float* _aidl_return) {
-         *_aidl_return = maxCpuValue();
+      ::ndk::ScopedAStatus ExampleService::GetMaxCpuTemperature(float* _aidl_return) {
+         *_aidl_return = MaxCpuValue();
          return ndk::ScopedAStatus::ok();
       }
 
-      ::ndk::ScopedAStatus ExampleService::getMaxGpuTemperature(float* _aidl_return) {
-         *_aidl_return = maxGpuValue();
+      ::ndk::ScopedAStatus ExampleService::GetMaxGpuTemperature(float* _aidl_return) {
+         *_aidl_return = MaxGpuValue();
          return ndk::ScopedAStatus::ok();
       }
 
-      ::ndk::ScopedAStatus ExampleService::getMaxAmbientTemperature(float* _aidl_return) {
-         *_aidl_return = maxAmbientValue();
+      ::ndk::ScopedAStatus ExampleService::GetMaxAmbientTemperature(float* _aidl_return) {
+         *_aidl_return = MaxAmbientValue();
          return ndk::ScopedAStatus::ok();
       }
 
-      void ExampleService::readColumnFromCSV(int colIndex, std::vector<float>& columnData) {
-         std::ifstream file;
-         file.open("/vendor/etc/temperatures.csv");
+      void ExampleService::ReadColumnFromCSV(int colIndex, std::vector<float>& columnData) {
+         std::ifstream file("/vendor/etc/temperatures.csv");
          std::string line;
 
          if (file.is_open()) {
```

Comment:
o chestie de stil, minor, dar eu prefer "early exit" atunci cand apare o eroare. Aplicand acest stil, vei evita functii ce au linii foarte lungi, cu indent mare.
e.g.
```suggestion
         if (!file.is_open()) {
              // error log.
              return;
         }
```